### ISSUE TITLE
confusion on install process

### ISSUE THREAD

**Issue by aculich** [dlab-berkeley/stata-fundamentals#13]
This [google doc](https://docs.google.com/document/d/13NiZTvSbd9I97M8srz4E1rMYA7Ugd4qGmdAflbdTR7I/edit) contains the instructions that we send to people ahead of the workshop.

Apparently there is some confusion from participants about these instructions. @melaniephillips can you reply to this thread and describe details about what confusion they had and what questions they were asking? Thanks!

**Comment by melaniephillips** [comment]
Hi All,

I think students just did not have all the information in one place which
caused confusion about how to download what from where. Moreso, that it
would have been ideal to have them set up and ready to go at the beginning
of the workshop. I think this document is great. We could also think about
filming step by step or add images to this document. Does this go out to
the students beforehand?

Best,

Melanie L. Phillips
PhD Candidate, Political Science
UC Berkeley
*pronouns: she/her/hers*

**Comment by aculich** [comment]
@melaniephillips yes, the instructions were emailed (via the calendar invite) to students ahead of time, but one of the things we should do is integrate the text of the instructions into the main [README.md](https://github.com/dlab-berkeley/stata-fundamentals/blob/master/README.md) in this repo, and then just distribute the license number and username/password info separately.
